GTViewController/Classes/Nav Classes/GTFollowupViewController.m

```diff
@@ -296,4 +270,55 @@ - (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)butto
     [alertView dismissWithClickedButtonIndex:buttonIndex animated:YES];
 }
 
+
+#pragma mark - UISnuffleButtonTapDelegate methods
+
+- (void)didReceiveTapOnPositiveButton:(UISnuffleButton *)positiveButton {    
+    NSArray *inputValidationErrors = [self inputValidationErrors];
+    
+    if ([positiveButton forceValidation] && [inputValidationErrors count]) {
```

Comment:
True, but it's good to recheck and not assume that another caller of this delegate added in the future has already checked.